internal/cmd/commands/credentialscmd/username_password_funcs.go

```diff
@@ -84,9 +84,9 @@ func (c *UsernamePasswordCommand) extraUsernamePasswordHelpFunc(_ map[string]fun
 
 	case "update":
 		helpStr = base.WrapForHelpText([]string{
-			"Usage: boundary credentials update user password [options] [args]",
+			"Usage: boundary credentials update username password [options] [args]",
```

Comment:
This is a mistake on my side when it was added as `user password` this should be `username-password [options]...`

Reply:
thats a good catch